pkg/virt-controller/services/template_test.go

```diff
@@ -501,6 +501,12 @@ var _ = Describe("Template", func() {
 				Expect(pod.Spec.Containers[1].Name).To(Equal("hook-sidecar-0"))
 				Expect(pod.Spec.Containers[1].Image).To(Equal("some-image:v1"))
 				Expect(pod.Spec.Containers[1].ImagePullPolicy).To(Equal(k8sv1.PullPolicy("IfNotPresent")))
+				Expect(pod.Spec.Containers[1].VolumeMounts[0].SubPath).To(Equal("hook-sidecar-0"))
```

Comment:
How about moving these assertions to `It("should call sidecar creators", func() {`? Since It is a dedicated sidecar test.